src/notso_glb/utils/gltfpack.py

```
@@ -0,0 +1,103 @@
+        cmd.extend(["-si", str(simplify_ratio)])
+
+    if texture_quality is not None:
+        texture_quality = int(texture_quality)
+        if not (1 <= texture_quality <= 10):
+            raise ValueError(
+                f"texture_quality must be in [1, 10], got {texture_quality}"
+            )
+        cmd.extend(["-tq", str(texture_quality)])
+
+    try:
+        result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)